index.d.ts

```diff
@@ -35,6 +35,8 @@ declare namespace albatross {
 
     deleteOne (filter: mongodb.FilterQuery<TSchema>, options?: mongodb.CommonOptions & { bypassDocumentValidation?: boolean }): Promise<0 | 1>
     deleteMany (filter: mongodb.FilterQuery<TSchema>, options?: mongodb.CommonOptions): Promise<number>
+
+    aggregate<T = TSchema> (pipeline: object[], options?: mongodb.CollectionAggregationOptions): Promise<any[]>
```

Comment:
Missing `T` in the return value

```suggestion
    aggregate<T = TSchema> (pipeline: object[], options?: mongodb.CollectionAggregationOptions): Promise<T[]>
```

Reply:
I wish that the official typings had typed `pipeline` 😅